app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<%
+=begin%>
+<h1>User Registration</h1>
+<form action='/users' method='POST'> <br>
+  <label for='username'>Username</label> <br>
```

Comment:
This should have the user object. More like => `<label for='user[username]'>Username</label>`